closure/goog/net/websocket.js

```diff
@@ -27,6 +27,7 @@
  */
 
 goog.provide('goog.net.WebSocket');
+goog.provide('goog.net.WebSocket.ClosedEvent');
```

Comment:
Just a heads up -- We will likely delete all of the goog.provides except for `goog.net.WebSocket` at some point in the future (probably within the next year. If you are requiring this in a `goog.provide` file, you should simply require `goog.require('goog.net.WebSocket')` to make it forward-compatible; if you're requiring this in a `goog.module` file, you would need to import it as `const {ClosedEvent} = goog.require('goog.net.WebSocket')`.

Reply:
As an additional heads up, we will actually be removing all of the provides (except for the first one) in this change as well.